src/assets/custom/css/main.scss

```diff
@@ -21,6 +22,7 @@ $vertical-overlap: 100px;
 @import "page-location-anchor";
 @import "section-intro";
 @import "service-line-teaser-group";
+@import "tabber";
```

Comment:
Should this be in alphabetical order?

Reply:
Good call. Fixed here: https://github.com/codurance/site/pull/1850/commits/49fe0d0bab134182875522e16c01bf5db783c059